### PR TITLE
fix logic that checks missing project configs

### PR DESCRIPTION
**Pull request by owlas**

### CODE REVIEW

File: packages/backend/src/config/parseConfig.mock.ts [hunk 1]
```diff
@@ -45,8 +45,10 @@ export const LOCAL_PROJECT_MISSING_PROJECT_DIR = {
 };
 
 export const LOCAL_PROJECT_MISSING_PROFILES_DIR = {
-    ...LOCAL_PROJECT,
-    profiles_dir: undefined,
+    type: 'dbt',
+    name: 'project',
+    profiles_dir: 'hello',
```
Comment (ZeRego) [inline packages/backend/src/config/parseConfig.mock.ts:50]: should this prop be removed ( based on the var name )

Comment (ZeRego) [reply packages/backend/src/config/parseConfig.mock.ts:50]: or the variable should be named `LOCAL_PROJECT_MISSING_PROJECT_DIR`

Comment (owlas) [reply packages/backend/src/config/parseConfig.mock.ts:50]: Yep missed something when merging these changes. Thanks!

**Comment by owlas** [comment]
@ZeRego updated! good to go?

**Review by ZeRego — APPROVED** [review]

**Pull request merged**